src/vaadin-context-menu-overlay.html

```diff
@@ -90,6 +94,31 @@
           });
         }
 
+        _themeChanged(theme) {
+          const childContextMenu = this.querySelector('vaadin-context-menu');
```

Comment:
Remove this logic. Theme changes, while the overlay is open, are very unlikely. To be on the safe side, we could close the overlay & empty the root element here instead.

Reply:
Replaced with closing the overlay & emptying the root element